build.gradle.kts

```diff
@@ -33,7 +33,7 @@ tasks {
 
     patchPluginXml {
         sinceBuild.set("221")
-        untilBuild.set("231.*")
+        untilBuild.set("232.*")
```

Comment:
I am wondering do we have to set untilBuild ? Because we will have to update this with each new release ?